istioctl/cmd/istioctl/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 
 	"istio.io/istio/istioctl/cmd/istioctl/gendeployment"
+        "istio.io/istio/istioctl/pkg/install"
```

Comment:
Do you mind running `gofmt` on those two files to fix those formatting issues?

Reply:
OK